gravitee-am-service/src/main/java/io/gravitee/am/service/impl/FormServiceImpl.java

```diff
@@ -297,6 +301,24 @@ public Completable delete(String domain, String formId, User principal) {
         return delete(ReferenceType.DOMAIN, domain, formId, principal);
     }
 
+    @Override
+    public Single<Form> getDefaultByDomainAndTemplate(String domain, String template) {
+        LOGGER.debug("Default form by domain {} and template {}", domain, template);
+
+        return Single.create(emitter -> {
+            try {
+                final String path = "views/default/" + template + ".html";
```

Comment:
Open question:

Should we introduce theme or domain in the directory tree to highlight the fact that these templates are the defaults coming from the gateway ?

Reply:
the `default` folder is created as the `view` folder contains a `login.html`. The sub-folder structure could be any appropriate directories. I have no reservation.